templates/client-daemonset.yaml

```diff
@@ -101,7 +101,7 @@ spec:
                 -advertise="${POD_IP}" \
                 -bind=0.0.0.0 \
                 -client=0.0.0.0 \
-                {{- if .Values.client.grpc }}
+                {{- if or .Values.client.grpc .Values.meshGateway.enabled }}
```

Comment:
gateways require gRPC. I can either do this or require `client.grpc` to be set to true. Requiring them to set this in client.grpc is cleaner because then they don't get grpc unexpectedly. But then we should find a way to require that if meshGateway.enabled is true, that client.grpc is true.

Reply:
Okay, I'll do that in a separate PR.

Reply:
I'm actually going to remove this because it feels wrong to set grpc to true automatically. Instead I'll document that it must be set to true for gateways to work *and* I've found a neat trick to have helm fail if these things aren't set properly. I'll tag you on it when it's committed.